zsh/.zgen-setup

```diff
@@ -7,12 +7,14 @@
 # BSD licensed, see LICENSE.txt
 
 # Clone zgen if you haven't already
-if [[ ! -f ~/zgen/zgen.zsh ]]; then
-  pushd ~
+ZGEN_PARENT_DIR=$HOME/github
```

Comment:
Please precede this with something like 
```
if [[ -z "$ZGEN_PARENT_DIR"]]; then
  ZGEN_PARENT_DIR=$HOME
fi
```
so it doesn't change behavior for existing users.